quines/README.md

```diff
@@ -0,0 +1,11 @@
+# Binaris Quine Collection
+
+A [quine](https://en.wikipedia.org/wiki/Quine_(computing)) is a non-empty computer program which takes no input and produces a copy of its own source code as its only output.  
+An [ouroboros](https://en.wikipedia.org/wiki/Ouroboros) is a serpent eating its own tail, and a recursion of the quine concept.
+
+You'll find here two quines, one in JavaScript, and one in Python, and "one" ouroboros in JavaScript and Python.  
```

Comment:
Maybe

"This repo contains two quines, one in JavaScript, and one in Python. It also contains  "one" ouroboros in JavaScript and Python."

Note that the subtle "one" joke/comment will probably be lost on readers who don't come in with entry knowledge.